src/analytics/private/sqls/redshift/segment-user.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.segment_user (
```

Comment:
What's the purpose of this table?

For unloading the data, could you only use a temporary table?

Reply:
This table persists user IDs for all the segments. We need this table for one of the segment filters, which is whether user is or is not in the specific segment.